### PR TITLE
Fix nextpnr interchange installed device

### PR DESCRIPTION
**Pull request by acomodi**
The installed devices do miss patches information, relevant for constraints and LUTs definition.

This PR installs the correct devices and also removes the symbiflow-yosys run dependency package which should not be needed.

### CODE REVIEW

File: pnr/nextpnr/fpga_interchange/build.sh
Comment (PiotrZierhoffer) [inline pnr/nextpnr/fpga_interchange/build.sh:39]: @acomodi do you track this in an issue somewhere?

**Review by PiotrZierhoffer — APPROVED** [review]

File: pnr/nextpnr/fpga_interchange/build.sh
Comment (acomodi) [reply pnr/nextpnr/fpga_interchange/build.sh:39]: Hmm not really, it's not really an issue, probably this should be a TODO rather than a FIXME

**Review by ajelinski — APPROVED** [review]
If you're going to change that `FIXME` @acomodi also please unify the tags in commit messages.

**Comment by acomodi** [comment]
> If you're going to change that `FIXME` @acomodi also please unify the tags in commit messages.

Done

**Comment by ajelinski** [comment]
@acomodi I missed the fact that if the package doesn't require `symbiflow-yosys` anymore then it doesn't need to wait for it in CI. So remove also this line from the workflow file with the `requirements/run` change: https://github.com/hdl/conda-eda/blob/277f77b52db710cfcfed5b9522cb09597a5867c1/.github/workflows/build-packages.yml#L367

After that I will merge the PR as none of these latest changes can cause a CI failure.

**Pull request merged**
